cs-jclouds/src/main/java/io/cloudslang/content/jclouds/entities/inputs/EbsInputs.java

```diff
@@ -0,0 +1,132 @@
+package io.cloudslang.content.jclouds.entities.inputs;
+
+import io.cloudslang.content.jclouds.entities.constants.Constants;
+import io.cloudslang.content.jclouds.utils.InputsUtil;
+
+/**
+ * Created by Mihai Tusa.
+ * 9/15/2016.
+ */
+public class EbsInputs {
+    private String blockDeviceMappingDeviceNamesString;
+    private String blockDeviceMappingVirtualNamesString;
+    private String deleteOnTerminationsString;
+    private String encryptedString;
+    private String iopsString;
+    private String snapshotIdsString;
+    private String volumeSizesString;
+    private String volumeTypesString;
+
+    private boolean ebsOptimized;
+
+    private EbsInputs(EbsInputs.EbsInputsBuilder builder) {
+        this.blockDeviceMappingDeviceNamesString = builder.blockDeviceMappingDeviceNamesString;
+        this.blockDeviceMappingVirtualNamesString = builder.blockDeviceMappingVirtualNamesString;
+        this.deleteOnTerminationsString = builder.deleteOnTerminationsString;
+        this.encryptedString = builder.encryptedString;
+        this.iopsString = builder.iopsString;
+        this.snapshotIdsString = builder.snapshotIdsString;
+        this.volumeSizesString = builder.volumeSizesString;
+        this.volumeTypesString = builder.volumeTypesString;
+
+        this.ebsOptimized = builder.ebsOptimized;
+    }
+
+    public String getBlockDeviceMappingDeviceNamesString() {
+        return blockDeviceMappingDeviceNamesString;
+    }
+
+    public String getBlockDeviceMappingVirtualNamesString() {
+        return blockDeviceMappingVirtualNamesString;
+    }
+
+    public String getDeleteOnTerminationsString() {
+        return deleteOnTerminationsString;
+    }
+
+    public String getEncryptedString() {
+        return encryptedString;
+    }
+
+    public String getIopsString() {
+        return iopsString;
+    }
+
+    public String getSnapshotIdsString() {
+        return snapshotIdsString;
+    }
+
+    public String getVolumeSizesString() {
+        return volumeSizesString;
+    }
+
+    public String getVolumeTypesString() {
+        return volumeTypesString;
+    }
+
+    public boolean isEbsOptimized() {
+        return ebsOptimized;
+    }
+
+    public static class EbsInputsBuilder {
+        private String blockDeviceMappingDeviceNamesString;
```

Comment:
maybe these can be final - depends whether you want to be able to do `withProp().withProp()`